### ISSUE TITLE
Need to add more wallets such as WalletConnect, Coinbase, Formatic, etc

### ISSUE THREAD

**Issue by best-lucky1030** [Shmoji/web3-react-example#3]
@Shmoji 
I read your medium article from here.
https://medium.com/coinmonks/web3-react-connect-users-to-metamask-or-any-wallet-from-your-frontend-241fd538ed39
I'd like to add more wallets such as Metamask, WalletConnect, Coinbase, etc like this image.
![image](https://user-images.githubusercontent.com/89365150/157432658-842b9abc-1b66-417c-9cf5-bd2c77ad6820.png)
I tried to find this, but couldn't do it.
Could you guide me for this?

Thanks,

**Comment by ramseynasr** [comment]
@best-lucky1030 I'm new to this world, and I just finished my first Web3 Smart Contract UI App for a client. While trying to figure out wallet connectivity, what I did come across, and I remember this in @Shmoji 's video tutorial and several other tutorials is: to add more wallets, you just need to import the injector from the @web3-react package that you want/need:

import { InjectedConnector } from "@web3-react/injected-connector"; //MetaMask we used this one.

Here are some other ones available at (https://github.com/NoahZinsmeister/web3-react)

![Screen Shot 2022-03-14 at 5 48 50 PM](https://user-images.githubusercontent.com/85218428/158284507-78d2a207-f756-40ef-a395-771bcc0c9749.png)

Wallet Connect uses [@web3-react/walletconnect](https://github.com/NoahZinsmeister/web3-react/blob/main/packages/walletconnect)

Postis uses https://openbase.com/js/@web3-react/portis-connector

Coinbase -> https://www.coinbase.com/cloud/discover/dev-foundations/use-web3-react-to-connect-wallet

Basically, just google the connector you are looking for "Name of Wallet @web3-react wallet connector" and it will show you a ton of results.

Then, and you probably meant this, you will have to design your own UI, modal, etc with those options. I am looking for a similar boilerplate, as I haven't found one myself. Once I have some time, I may write one and throw it on my Git for everyone to use as a starting point. I just wish someone else already offered this  because I hate reinventing the wheel. And its just a simple modal... lol

**Comment by best-lucky1030** [comment]
@ramseynasr 
Thanks for your suggestion.
I will look around another repo. Maybe @Shmoji 's video will help me.
